package.json

```diff
@@ -5,7 +5,8 @@
   "main": "bitCoinCmd.js",
   "dependencies": {
     "concat-stream": "^1.5.0",
-    "promise": "^7.0.4"
+    "promise": "^7.0.4",
```

Comment:
Might be easier to use `bluebird`, that's the promise library used by `request-promise`?